content/embeds/shard-placement-intro.md

```diff
@@ -0,0 +1,17 @@
+In Redis Enterprise Software (RS), the location of master and slave shards on the cluster nodes can impact the database and node performance.
+Master shards are always hosted separately from their slave shards.
+By default, all master shards for a database are hosted on the same node and all slave shards are hosted on a different node,
+as long as the nodes have enough resources to host them.
```

Comment:
This is very misleading.
1. I wouldn't mention the default, but rather state that there is a shards_placement policy per bdb.
2. What you describe is the 'dense' policy. The 'sparse' should also be mentioned.
3. As this is only a policy, there is no guarantee that the (same role) shards will end up on the same node, since there are other considerations in placement planning.

Reply:
I rewrote it based on your comments.